### ISSUE TITLE
How to merge all scenaries in one feature?

### ISSUE THREAD

**Issue by i-vasilev** [Invictum/serenity-reportportal-integration#142]
I have one feature file. It has a few scenaries. But after running report portal divides feature for scenaries. So, I have a few the same features in the ReportPortal list. Is it your issue? Or maybe, I need to go to ReportPortal developers?
![image](https://user-images.githubusercontent.com/17360663/129731105-362bd9e5-584a-4957-9963-7b631e7390c0.png)

**Comment by Invictum** [comment]
@i-vasilev 
By design each scenario in feature file will produce separate test result in RP.
Can you list your feature file?

**Comment by i-vasilev** [comment]
@Invictum This is the list my feature file:
![image](https://user-images.githubusercontent.com/17360663/130571901-71161b7f-32ce-4e54-a912-3eba492bb656.png)

I think the scenaries should be in one section in RP

**Comment by Invictum** [comment]
Yes for sure, it should produce one test in RP. It looks like a bug in integration.
My goal is to finish version 2.0 where most of issues will be resolved. So keep an eye on it.

**Comment by i-vasilev** [comment]
@Invictum Hello! When will you approximately finish version 2.0?

**Comment by i-vasilev** [comment]
@Invictum 
I've found this is not a problem:
My test reruned but I put `failsafe.rerunFailingTestsCount` in reportportal.properties. Please, add annotation to your Readme where we should write the property.
